ogr/abstract.py

```diff
@@ -1394,3 +1394,20 @@ def get_projects(self) -> Sequence["GitProject"]:
 
     def get_forks(self) -> Sequence["GitProject"]:
         raise NotImplementedError
+
+
+class GithubTokenManager:
+    """
+    Represents a token manager for authentication via GitHubApp.
+    """
+
+    def get_instance(self, namespace: str, repo: str) -> Any:
+        """
+        Returns GitHub instance for a requested repository,
```

Comment:
The term `instance` is a bit misleading -- would be nice to explicitly state that we are talking about Github instance from pygithub.